backend/app/services/work_queue_service.py

```
@@ -593,22 +636,196 @@ def ensure_default_queue(self, user_id: int) -> Kind:
+            if existing_queue:
+                return existing_queue, False
+            # Fallback: try to get inbox by name
+            with self.get_db() as db:
+                inbox = (
+                    db.query(Kind)
+                    .filter(
+                        Kind.user_id == user_id,
+                        Kind.kind == self.WORK_QUEUE_KIND,
+                        Kind.namespace == "default",
+                        Kind.name == "inbox",
+                        Kind.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)